examples/dataflow-data-generator/data-generator-pipeline/tests/test_data_generator.py

```diff
@@ -126,10 +123,6 @@ def test_generate_fake(self):
         # Check float strictly positive
         self.assertGreaterEqual(actual_row[u'lo_tax'], 0.0)
```

Comment:
Is there a reason for unicode string comparison?

Reply:
I use unicode strings because that's what the `json` library does by default.